content/references/protocol-reference/data-types/nftoken.md

```diff
@@ -112,36 +112,36 @@ The example sets three flags: lsfBurnable (0x0001), lsfOnlyXRP (0x0002), lsfTran
 
 ### TransferFee
 
-The TransferFee value specifies the percentage fee, in units of 1/10,000, charged by the issuer for secondary sales of the token. Valid values for this field are between 0 and 50,000, inclusive. A value of 1 is equivalent to 1bps or 0.01%, allowing transfer rates between 0% and 50%.
+The `TransferFee` value specifies the percentage fee, in units of 1/10,000, charged by the issuer for secondary sales of the token. Valid values for this field are between 0 and 50,000, inclusive. A value of 1 is equivalent to 1bps or 0.00001%, allowing transfer rates between 0% and 50%.
 
 
 ### Example
 
-This value sets the transfer fee to 314 bps, or 3.14%.
+This value sets the transfer fee to 314, or .00314%.
 
 ![Txr Fee](img/nftokenb.png "Txr Fee")
 
 
 
 ### Issuer Identification
 
-The third section of the TokenID is a big endian representation of the issuer’s public address.
+The third section of the `NFTokenID` is a big endian representation of the issuer’s public address.
 
 ![Issuer Address](img/nftokenc.png "Issuer Address")
 
 
 
-### Taxon
+### NFTokenTaxon
 
-The fourth section is a taxon created by the issuer.
+The fourth section is a `NFTokenTaxon` created by the issuer.
 
-![Taxon](img/nftokend.png "Taxon")
+![NFTokenTaxon](img/nftokend.png "NFTokenTaxon")
 
-An issuer might issue several NFTs with the same taxon; to ensure that NFTs are spread across multiple pages, the taxon is scrambled using the fifth section, a dumb sequential number, as the seed for a random number generator. The scrambled value is stored with the `NFToken`, but the unscrambled value is the actual taxon.
+An issuer might issue several `NFToken` objects with the same `NFTokenTaxon`; to ensure that `NFToken` objects are spread across multiple pages, the `NFTokenTaxon` is scrambled using the fifth section, a dumb sequential number, as the seed for a random number generator. The scrambled value is stored with the `NFToken`, but the unscrambled value is the actual NFTokenTaxon.
```

Comment:
I suggest striking the word "dumb" from this sentence, since it's not necessary and can be an emotionally-charged word. Also, I noticed an inconsistent use of `code font` for `NFTokenTaxon` here.

```suggestion
An issuer might issue several `NFToken` objects with the same `NFTokenTaxon`; to ensure that `NFToken` objects are spread across multiple pages, the `NFTokenTaxon` is scrambled using the fifth section, a sequential number, as the seed for a random number generator. The scrambled value is stored with the `NFToken`, but the unscrambled value is the actual `NFTokenTaxon`.
```